app/upgrade_handler.go

```diff
@@ -13,21 +13,22 @@ import (
 )
 
 const (
-	upgradeName = "v2.7.0"
+	upgradeName = "v2.8.0"
```

Comment:
It seems to be "v2.7.0" on the current master.

Reply:
2.8.0 for next upgrade, including bounty module